lib/rucio/daemons/conveyor/poller.py

```diff
@@ -234,6 +234,19 @@ def poll_transfers(external_host, xfers, prepend_str='', request_ids=None, timeo
             record_timer('daemons.conveyor.poller.bulk_query_transfers', (time.time() - tss) * 1000 / len(xfers))
         except (TransferToolTimeout, TransferToolWrongAnswer) as error:
             logging.error(prepend_str + str(error))
+            if isinstance(error, TransferToolWrongAnswer):
```

Comment:
Shouldn't this be a separate exception handler instead of testing with `isinstance`?

I'm also not entirely convinced to have processing logic in the exception handler, can this be done differently?

Reply:
Different handler would be better, makes it easier readable and the code duplication is minimal (logging.error twice)